statsmodels/genmod/generalized_linear_model.py

```diff
@@ -1051,7 +1064,8 @@ def __init__(self, model, params, normalized_cov_params, scale,
         self._endog = model.endog
         self.nobs = model.endog.shape[0]
         self.mu = model.mu
-        self._data_weights = model.data_weights
+        # Divide by n for binom
+        self._freq_weights = model.freq_weights / model.n
```

Comment:
This looks fragile/dangerous to me. We are mixing up the two meanings of weights in binomial.
My guess would be that this shouldn't be divided by `model.n`